org.sugarlabs.MazeActivity.json

```diff
@@ -0,0 +1,269 @@
+{
+    "app-id": "org.sugarlabs.MazeActivity",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "separate-locales": false,
+    "build-options": {
```

Comment:
Move this to the specific modules where it is used, you can copy what I do here https://github.com/tchx84/org.sugarlabs.Chart-flathub/blob/master/org.sugarlabs.Chart.json

Reply:
469baad